pkg/storaged/stratis/pool.jsx

```diff
@@ -64,6 +65,283 @@ const _ = cockpit.gettext;
 const fsys_min_size = 512 * 1024 * 1024;
 const fsys_round_size = 1024 * 1024;
 
+/* Abstractions over the r6 and r8 API revisions, and V1 and V2 pool
+ * formats.
+ */
+
+/* Key descriptions are returned as an array with elements of type
+ *
+ * { slot: number | null, keydesc: string }
+ *
+ * A V1 pool will have at most one entry in the array, and it will
+ * have `slot == null`.  V2 pools have zero or more entries with
+ * numbered slots.
+ */
+
+function get_key_descriptions(pool) {
+    const result = [];
+
+    if (!pool.Encrypted)
+        return result;
+
+    if (client.stratis_interface_revision < 8) {
+        const key_desc = pool.KeyDescription;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21927-df19b234-20250826-063853-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)